### DISCUSSION TITLE
Submission deadline

### DISCUSSION THREAD

**Discussion by checkdgt** [skhatiri/UAV-Testing-Competition#7] [Submission] [answered]
Hi, just wanted to confirm the deadline.
The email mentions the 6th but the README shows the 7th, can you confirm which it is please?
Thanks.
<img width="572" alt="Screenshot 2023-12-05 at 09 40 18" src="https://github.com/skhatiri/UAV-Testing-Competition/assets/8478412/becf0273-7c28-4b83-a90d-6752d621a942">

**Comment by skhatiri** [comment, marked as answer]
Hi, 
Thanks for mentioning this.
Always consider the repository as the final word, so 7.12.2023 AoE is the deadline.
